### DISCUSSION TITLE
fastapi v.0.63.0 does not support starlette latest versions

### DISCUSSION THREAD

**Discussion by ghost** [fastapi/fastapi#3056] [Questions] [answered]
Hello fastapi community,
I m new to fastapi and I just noticed that **_fastapi v0.63.0_** does not support **starlette v0.13.7+**

1-I m gettin an error from pip after installation as below,
**
**ERROR:** pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
fastapi 0.63.0 requires starlette==0.13.6, but you have starlette 0.14.2 which is incompatible.
**
2- after installing latest verstion of starlette when I try 
"uvicorn main:app --reload"  
I get an error as below
**
**ImportError:** cannot import name 'UJSONResponse' from 'starlette.responses' (/Users/tuncovacik/Documents/fastap_mongo/mongodb-with-fastapi/venv_mongofast/lib/python3.9/site-packages/starlette/responses.py)
**
I wonder if there is going to be a release to support latest versions of starlette?

thank you

**Comment by jcwilson** [comment, marked as answer]
There is this PR that should resolve your issue (#2335).

I'm not sure what it'll take to land it though. It's been open since November and stable for a month with no feedback from @tiangolo. Maybe he'll see this and take a look. :)

For now, you should probably pin starlette to 0.13.6 if possible (or not pin at all and let pip handle it for you).
